setup.py

```diff
@@ -706,10 +706,6 @@ def get_ext_modules() -> List[Extension]:
         "executorch/devtools/bundled_program": "devtools/bundled_program",
         "executorch/runtime": "runtime",
         "executorch/util": "util",
-        # Note: This will install a top-level module called "serializer",
-        # which seems too generic and might conflict with other pip packages.
-        "serializer": "backends/arm/third-party/serialization_lib/python/serializer",
```

Comment:
This is OK. I guess we have to do to two step process anyways, with `setup.sh` as a separate step.

Another option I am thinking was to overload `install_requirements.py` but not much material difference if we still have to run setup.sh. At least if we can do everything AoT with just that then it might be better. What do you think?

Reply:
I was thinking to overload ET's `install_requirements.py` script to optionally do `pip install git+https...` for Ethos setups. That way, if we could, at least for AoT-only experiments it can be a one step process, and you don't have to run setup.sh. 

If you want to do e2e i.e. install baremetal gcc, and FVP etc., I don't see a way to do it with just pip install today, and we may have to call a setup.sh from ET's `install_requirements.py` too. Then we can be a truly single step.


I don't have a strong preference on a shell script vs. a python script, but yeah let's keep it something which is not confusing i.e. not setup.py or install_requirements.py :-p

Reply:
Ok, then I think we are on the same page! Tbh I also prefer calling it something else than install_requirements.py, I think I misunderstood you hah!

Reply:
so we will do that as a different step?